pkg/util/k8sutil/k8sutil.go

```diff
@@ -196,6 +196,9 @@ func NewMemberServiceManifest(etcdName, clusterName string, owner metatypes.Owne
 		ObjectMeta: v1.ObjectMeta{
 			Name: etcdName,
 			Labels: map[string]string{
+				// Member service is different from etcd (client) service.
+				// The owner of member service is the pod itself, so we can't use "app=etcd" which is used by GC.
+				"app":          "etcd-member-svc",
```

Comment:
Without this label, member service is a "subset" of etcd (client) service from labels point of view. For example, if I only want to select member services, there is no way to do it.